packages/react-google-maps-api/src/components/drawing/Rectangle.tsx

```diff
@@ -49,28 +49,35 @@ interface RectangleState {
 }
 
 interface RectangleProps {
-  options: google.maps.RectangleOptions;
-  bounds: Bounds;
-  draggable: boolean;
-  editable: boolean;
-  visible: boolean;
-  onDblClick: (e: MouseEvent) => void;
-  onDragEnd: (e: MouseEvent) => void;
-  onDragStart: (e: MouseEvent) => void;
-  onMouseDown: (e: MouseEvent) => void;
-  onMouseMove: (e: MouseEvent) => void;
-  onMouseOut: (e: MouseEvent) => void;
-  onMouseOver: (e: MouseEvent) => void;
-  onMouseUp: (e: MouseEvent) => void;
-  onRightClick: (e: MouseEvent) => void;
-  onClick: (e: MouseEvent) => void;
-  onDrag: (e: MouseEvent) => void;
-  onBoundsChanged: () => void;
+  options?: google.maps.RectangleOptions;
+  bounds?: Bounds;
+  draggable?: boolean;
+  editable?: boolean;
+  visible?: boolean;
+  clickable?: boolean;
+  onDblClick?: (e: MouseEvent) => void;
+  onDragEnd?: (e: MouseEvent) => void;
+  onDragStart?: (e: MouseEvent) => void;
+  onMouseDown?: (e: MouseEvent) => void;
+  onMouseMove?: (e: MouseEvent) => void;
+  onMouseOut?: (e: MouseEvent) => void;
+  onMouseOver?: (e: MouseEvent) => void;
+  onMouseUp?: (e: MouseEvent) => void;
+  onRightClick?: (e: MouseEvent) => void;
+  onClick?: (e: MouseEvent) => void;
+  onDrag?: (e: MouseEvent) => void;
+  onBoundsChanged?: () => void;
 }
 
 export class Rectangle extends PureComponent<RectangleProps, RectangleState> {
   static contextType = MapContext
 
+  static defaultProps: Partial<RectangleProps> = {
```

Comment:
Uhh, I don't get it. Why would you turn every prop optional in the declaration above and then used the `Partial` here? It's the same thing :)

Reply:
I turned them optional because they were required by accident. What you're saying is that if they are all optional there's no need for the Partial right? And I think you are right :-)